test/cases/manifest_tests

```diff
@@ -110,7 +110,7 @@ def checkout_images_repo(ref, workdir: os.PathLike) -> str:
         print(f"Checking out '{OSBUILD_IMAGES_REPO_URL}' repository at ref '{ref}'")
         try:
             subprocess.check_call(
-                ["git", "clone", "--depth=1", "--no-single-branch", OSBUILD_IMAGES_REPO_URL, "images"],
+                ["git", "clone", "--depth=1", "--", f"revision={ref}", OSBUILD_IMAGES_REPO_URL, "images"],
```

Comment:
This should be
```suggestion
                ["git", "clone", "--depth=1", f"--revision={ref}", OSBUILD_IMAGES_REPO_URL, "images"],
```

Reply:
Ah, I copy/pasted it directly from your chat message. The command *does* do the right thing even in this format :>

Reply:
Done.

Reply:
That's odd, because I think that would translate to
```
git clone --depth=1 -- revision=<ref> <repo>
```
:thinking: